### ISSUE TITLE
Rank labels and file labels to be separated from board

### ISSUE THREAD

**Issue by sidnc86** [techvision/axchessible#6]
As rank labels and file labels are additional column or row in existing table element of board, screen reader announce board dimensions incorrectly. That is board appears to be 9x9 board where first column and last row cannot be navigated to.
Need to change the logic in a way that board stays a table of 8x8 grid and rank labels and file labels are placed around the board as separate containers.
